### DISCUSSION TITLE
TurboACC的新选项

### DISCUSSION THREAD

**Discussion by jh0072110** [immortalwrt/immortalwrt#827] [Q&A]
发现在fullconeNAT的选项里除了基础的xt_FULLCONENAT以外新增了Boardcom FullCone选项，请问…有什么区别…咳咳咳

**Comment by 1715173329** [comment]
没什么用

**Reply by jh0072110** [reply]
> 没什么用

谢谢大佬，那么…x86应该选哪个呢…还是都一样……

**Reply by 1715173329** [reply]
默认就行，跑 pcdn 就用 brcm
